examples/amp-story/interactive_slider.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html ⚡ lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <meta name="description" content="This is the AMP Boilerplate." />
+    <link rel="preload" as="script" href="https://cdn.ampproject.org/v0.js" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link
+      rel="preload"
+      as="script"
+      href="https://cdn.ampproject.org/v0/amp-story-1.0.js"
+    />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+      async
+      custom-element="amp-story"
+      src="https://cdn.ampproject.org/v0/amp-story-1.0.js"
+    ></script>
+    <script
+      async
+      custom-element="amp-story-interactive"
+      src="https://cdn.ampproject.org/v0/amp-story-interactive-0.1.js"
+    ></script>
+    <!-- Import other AMP Extensions here -->
+    <style amp-custom>
+      /* Add your styles here */
+      .title {
+        background-color: white;
+        border-radius: 60px;
+        padding: 40px;
+      }
+      h1 {
+        color: black;
+        font-size: 30px;
+        margin-bottom: 10px;
+        text-align: center;
+        font-family: 'Poppins';
```

Comment:
Since we're using this font globally, we could declare this in a body block like:
```
body {
  font-family: 'Poppins';
}
```

Then remove it in the `h1` and `p` blocks.